src/arguments/api_test.py

```diff
@@ -25,48 +29,102 @@ def fetch_input_url(cls):
         # Check if http:// or https:// is present in request_url
         has_protocol = cls.__check_protocol(request_url)
 
-        if not(has_protocol):
+        if not (has_protocol):
             request_url = "https://" + request_url
 
         # Ask the user for endpoint if not present in request_url
-        if not(has_endpoint):
-            if(request_url[-1] == '/'):
-                endpoint = input("Input endpoint " +
-                                 "(Without the starting slash): ")
+        if not (has_endpoint):
+            if request_url[-1] == "/":
+                endpoint = input("Input endpoint " + "(Without the starting slash): ")
             else:
                 endpoint = input("Input endpoint (With the starting slash): ")
             request_url += endpoint
 
         print("Trying ...\u26A1")
         return {
-            "request_url" : request_url,
-            "request_headers" : request_headers,
+            "request_url": request_url,
+            "request_headers": request_headers,
         }
 
-    #saves the json response into a file
     @classmethod
-    def save_response_data(cls,response_data):
-        store_data = input('Store response data? (Y/N): ')
-        if(store_data.lower() == 'y'):
-                filename = input("Enter a filename (response_data.json)")
-                if filename == '':
-                    filename = 'response_data.json'
-                with open(filename, 'w') as jsonFile:
-                    json.dump(response_data, jsonFile, indent=4)
-                print(f"Response data stored in {filename}")
-        elif(store_data.lower()) == 'n':
+    def read_data_from_file(cls):
+        filename = input("Enter a filename (response_data.json)")
+        data = {}
+        if filename == "":
+            filename = "response_data.json"
```

Comment:
```py
filename.strip()
```

Reply:
yeah sure, I will include it in my latest commit

Reply:
Thanks 😄